api4/plugin.go

```diff
@@ -100,22 +100,84 @@ func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	downloadUrl := r.URL.Query().Get("plugin_download_url")
+	pluginDownloadUrl := r.URL.Query().Get(model.PLUGIN_URL_STRING)
+	signatureDownloadUrl := r.URL.Query().Get(model.SIGNATURE_URL_STRING)
+	downloadUrls := map[string]string{
+		model.PLUGIN_URL_STRING: pluginDownloadUrl,
+	}
+	if signatureDownloadUrl != "" {
+		downloadUrls[model.SIGNATURE_URL_STRING] = signatureDownloadUrl
+	}
+
+	downloadedFiles := downloadFiles(c, downloadUrls)
+	pluginFile := downloadedFiles[model.PLUGIN_URL_STRING]
+	if pluginFile == nil {
+		return
+	}
+
+	if signatureDownloadUrl != "" {
```

Comment:
Does it make sense to add an extra parameter to indicate if it should `enforceVerification` and not only rely on having or not the signature file which can cause to skip the whole verification process?

Reply:
We talked about it with @lieut-data and decided that for the MVP we do not need this additional parameter.